linux_os/guide/services/ntp/service_timesyncd_configured/bash/shared.sh

```diff
@@ -0,0 +1,27 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+
+{{{ bash_instantiate_variables("var_multiple_time_servers") }}}
+
+IFS=',' read -r -a time_servers_array <<< "$var_multiple_time_servers"
+preferred_ntp_servers_array=("${time_servers_array[@]:0:2}")
+preferred_ntp_servers=$( echo "${preferred_ntp_servers_array[@]}"|sed -e 's/\s\+/,/g' )
+fallback_ntp_servers_array=("${time_servers_array[@]:2}")
+fallback_ntp_servers=$( echo "${fallback_ntp_servers_array[@]}"|sed -e 's/\s\+/,/g' )
+
+config_file="/etc/systemd/timesyncd.d/oscap-remedy.conf"
+current_cfg_arr=( "/etc/systemd/timesyncd.conf" )
+current_cfg_arr+=("$(ls /etc/systemd/timesyncd.d/*)")
+# Comment existing NTP FallbackNTP and RootDistance settings
+for current_cfg in "${current_cfg_arr[@]}"
+do
+    sed -i 's/^NTP/#&/g' "$current_cfg"
+    sed -i 's/^FallbackNTP/#&/g' "$current_cfg"
+done
+# Set primary fallback NTP servers and RootDistance in drop-in configuration
```

Comment:
This comment doesn't seem updated. Could you remove the reference for `RootDistance`, please?